README.md

```diff
@@ -41,14 +41,30 @@ Run it locally and read its output:
 $ pdd --help
 ```
 
-You can exclude certain files from the search, for example:
+### File and Directory Selection
```

Comment:
@Doldrums maybe it's better to keep the text you just deleted? Now it looks weird: you start a section without any text, just some example of code. It doesn't read well.

Reply:
okay, returned this text 😊